infra/general/KCurlWrapper.class.php

```diff
@@ -225,7 +225,7 @@ public function __construct($params = null) {
 		if(!$params || !isset($params->curlVerifySSL) || !$params->curlVerifySSL)
 		{
 			curl_setopt($this->ch, CURLOPT_SSL_VERIFYPEER, false);
-			curl_setopt($this->ch, CURLOPT_SSL_VERIFYHOST, 1);
+			curl_setopt($this->ch, CURLOPT_SSL_VERIFYHOST, 2);
```

Comment:
Since our server in production run older version of curl - 7.19.4,
and to maintain backward compatibility, lets add static function getSslVerifyHostValue() that will decide its value based on curl_version().